tests/integration/tier0/non-destructive/file-backup/test_file_backup.py

```diff
@@ -3,90 +3,151 @@
 import pytest
 
 
+MODIFIED_CONTENT = """\n#This is just a placeholder test
+#to verify the file won't be changed
+# after the rollback"""
+PACKAGES = {
+    "cloud-init": "/etc/cloud/cloud.cfg",
+    "NetworkManager": "/etc/NetworkManager/NetworkManager.conf",
+    "yum": "/etc/logrotate.d/yum",
+    "dnf": "/etc/logrotate.d/dnf",
+}
+BACKUP_DIR = "/tmp/c2r_tests_backup/"
+MODIFIED_FILES_DIR = "/tmp/c2r_tests_backup/modified/"
+
+
 @pytest.fixture
-def config_files(shell):
+def config_files_modified(shell):
     """
-    This fixture either modifies contents or removes completely two
-    configuration files (cloud-init, NetworkManager).
-    The action is based on test-related custom envar.
+    This fixture modifies contents of configuration files: "/etc/cloud/cloud.cfg",
+    "/etc/NetworkManager/NetworkManager.conf", "/etc/logrotate.d/yum"
+    and "/etc/logrotate.d/dnf" prior to running the conversion.
+    After the rollback the fixture validates that the contents of the file is
+    the same before and after the convert2rhel run.
+    Additionally, here was a clash happening in the way convert2rhel backups files,
+    if there is a file to back up that has the same name as one directory
+    or another file already created, an error will be thrown to the user.
+    The fixture validates this won't happen anymore.
```

Comment:
I'm not sure what the file clash here means, and did not find it in the fixture. Maybe we can sync about that in the office :D

Reply:
oh, that's just a typo
> Additionally, there was a clash ...

Reply:
Also, to give a bit more context, this is what was happening: 

```
/etc/logrotate.d/yum # This is a file
/var/lib/convert2rhel/backup/yum # This was a folder
```

Reply:
@r0x0d thanks for the explanation